README.md

```diff
@@ -16,4 +16,20 @@ docker run -e CRAPI_URL=http://host.docker.internal:8090 -e HTTP_PROXY=http://ho
 ./gradlew buildDockerImage
 export CRAPI_URL=<crapi url>
 docker-compose up
-```
\ No newline at end of file
+```
+
+
+
+**Step By Step Cypress Installation**
+1. Install Node.js. After Installation:
+  ## In MAC: 
+          Go to Terminal, echo $PATH, /usr/local/bin should be present in the path. If not present, vim ~/.bash_profile and click enter
```

Comment:
Please enclose the commands within backticks (`) (below the escape key on the keyboard). Backticks are used in markdown to highlight commands.

Reply:
Resolved